LICENSE.txt

```diff
@@ -0,0 +1,357 @@
+This program is free software; you can redistribute it and/or modify
```

Comment:
`MIGHT` - IIRC, we're _technically_ supposed to have the full file, but it feels much cleaner to just make the `License` section of the `README` link to https://www.gnu.org/licenses/old-licenses/gpl-2.0.html? That feels much cleaner.

IIRC, GitHub also has something where you can mark the license, and it'll show up in GH's interface somewhere.